# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pyrequests/certs.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/requests/certs.py b/requests/certs.py
index d1a378d7..4e0bffd4 100644
--- a/requests/certs.py
+++ b/requests/certs.py
@@ -12,7 +12,8 @@ If you are packaging Requests, e.g., for a Linux distribution or a managed
 environment, you can change the definition of where() to return a separately
 packaged CA bundle.
 """
-from certifi import where
+def where():
+    return "/etc/ssl/certs/ca-certificates.crt"
 
 if __name__ == '__main__':
     print(where())
